common/src/main/scala/aws/S3.scala

```diff
@@ -0,0 +1,72 @@
+package aws
+
+import java.io.{ByteArrayInputStream, InputStream}
+
+import com.amazonaws.services.s3.AmazonS3
+import com.amazonaws.services.s3.model.{ObjectMetadata, PutObjectRequest, PutObjectResult, S3Object}
+import com.amazonaws.util.IOUtils
+import exception.TopicCounterException
+import models.TopicCount
+import org.slf4j.{Logger, LoggerFactory}
+import play.api.libs.json.{Format, JsError, JsSuccess, Json}
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.{Failure, Success, Try}
+
+trait S3[T]  {
+
+  def s3Client: AmazonS3
+  def bucketName: String
+  def path: String
+  def logger: Logger
+
+  def put(data: Seq[T])(implicit format: Format[T]) : PutObjectResult =  {
```

Comment:
do we put anything?

Reply:
Yes, this is the class that the lambda uses to write the  topic counts to s3. Further down you can see the file being deleted from the workerlambda project. Github is confused because I've added to  it as well as moved it

Reply:
Got it 👍